lessons/PlottinginPythonwithMatplotlib/example.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# moviewriter.py
+import numpy as np
+import pylab as p
```

Comment:
Please don't encourage use of `pylab`. It is a) deprecated and b) confusing for new users.